core/templates/dev/head/pages/exploration_player/HintAndSolutionModalService.js

```diff
@@ -17,17 +17,19 @@
  */
 
 oppia.factory('HintAndSolutionModalService', [
-  '$uibModal', 'UrlInterpolationService', 'HintsAndSolutionManagerService',
-  'PlayerPositionService',
-  'AudioTranslationManagerService', 'AudioPlayerService',
-  'EVENT_AUTOPLAY_AUDIO', 'COMPONENT_NAME_HINT', 'PlayerTranscriptService',
-  'COMPONENT_NAME_SOLUTION', 'AutogeneratedAudioPlayerService',
+  '$uibModal', 'AudioTranslationManagerService',
+  'AutogeneratedAudioPlayerService',
+  'HintsAndSolutionManagerService',
+  'PlayerPositionService', 'PlayerTranscriptService',
+  'ServiceAudioPlayerService', 'UrlInterpolationService', 'COMPONENT_NAME_HINT',
```

Comment:
@anubhavsinha98, `ServiceAudioPlayerService` do we need this? (also, do we have any service like this?)

How I found this:
- You'll find a link for each error in angularJS error which will explain what's going wrong, I found this [link](https://code.angularjs.org/1.5.8/docs/error/$injector/unpr?p0=ServiceAudioPlayerServiceProvider%20%3C-%20ServiceAudioPlayerService%20%3C-%20HintAndSolutionModalService) over [here](https://travis-ci.org/oppia/oppia/jobs/480758947#L5418).

Reply:
Oh really thanks @DubeySandeep , actually it's my fault!